wiki/osu!_originals/es.md

```diff
@@ -199,6 +199,12 @@ Estas canciones son parte de [osu! James Landino Collection EP](https://fanlink.
 | :-: | :-: | :-- |
 | [aa...hisuichazuke... - steganography](https://www.youtube.com/watch?v=CRD-2sk6Zt8)[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/1846639) | Desempate de la gran final |
 
+#### ![](/wiki/shared/mode/mania.png) [osu!mania 7K World Cup 2023](/wiki/Tournaments/MWC/2023_7K)
+
+| Canción | Beatmap | Notas |
+| :-: | :-: | :-- |
+| Toromaru - Uncharted Sky[^fa] | [#1](https://osu.ppy.sh/beatmapsets/1931471) | Rice 6 de semifinales |
```

Comment:
Would use "Semifinales RC6" to use both the abbreviation suggested in the main comment and having it consistent with previous translations from OWC.